### DISCUSSION TITLE
buddhism/maranasati/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#10] [General]
# buddhism/maranasati/

https://shanenull.com/buddhism/maranasati/

**Comment by shane0** [comment]
https://youtube.com/clip/UgkxPDMd9M_qYSZ6Cjm36BSw39xEoHZF0qpR?si=_SF9yEcrFxEccMD7

**Comment by shane0** [comment]
https://youtu.be/7r2i4Z3sQDc

**Comment by shane0** [comment]
**Maranasati in the style of Cormac McCarthy:**

The dead know not of aimlessness or struggle.

They are beyond such things.

For them, there is only the stillness.

The stillness of the grave.

The stillness of the void.

Maranasati is the meditation on death.

It is the remembrance of the impermanence of all things.

It is the contemplation of our own mortality.

Maranasati is not a grim practice.

It is a practice of liberation.

For when we are truly aware of our own mortality, we are free to live our lives to the fullest.

We are free to let go of our attachments and our fears.

We are free to simply be.

In the face of death, all else fades away.

Our petty concerns and our meaningless pursuits lose their significance.

We are left with only the essentials.

We are left with only the moment.

Maranasati is a practice of radical honesty.

It is a practice of facing the truth of our own existence.

It is a practice of accepting the inevitability of our own deaths.

And it is a practice of finding freedom in that acceptance.

So let us meditate on death.

Let us remember that we are all going to die.

And let us use that knowledge to live our lives to the fullest.

**Comment by shane0** [comment]
# common deaths

- nlp generated

- average lifespan in usa is 76
- 6 out of 10 people in the United States will die from cancer or heart complications.
- 4 out of 10 people in the US will get cancer
- 4 out of 10 people in the US will develop heart complications
- diabetes shortens life on average 8.5 years

## cancer

Sure. Here is a breakdown of the types, genders, percentages, and age of cancer cases in the United States:

**Types of cancer**

| Type of cancer | Percentage of all cancer cases |
|---|---|
| Breast cancer | 30.8% |
| Prostate cancer | 14.1% |
| Lung cancer | 12.9% |
| Colorectal cancer | 10.9% |
| Melanoma | 7.6% |
| Other types of cancer | 23.7% |

**Genders**

| Gender | Percentage of all cancer cases |
|---|---|
| Male | 50.3% |
| Female | 49.7% |

**Percentages**

| Age group | Percentage of all cancer cases |
|---|---|
| 0-19 years | 0.4% |
| 20-39 years | 8.5% |
| 40-59 years | 26.1% |
| 60-79 years | 36.2% |
| 80 years or older | 28.8% |

**Note:** These percentages are based on data from the American Cancer Society and reflect the estimated number of new cancer cases in 2023.

## How to reduce your risk of cancer

There are a number of things you can do to reduce your risk of cancer, including:

* Quitting smoking
* Eating a healthy diet
* Exercising regularly
* Maintaining a healthy weight
* Protecting yourself from the sun
* Getting regular cancer screenings

If you have any concerns about your risk of cancer, talk to your doctor. They can help you assess your risk and develop a plan to reduce your risk.

## heart

Sure. Here is a breakdown of the types, genders, percentages, and age of heart disease cases in the United States:

**Types of heart disease**

| Type of heart disease | Percentage of all heart disease cases |
|---|---|
| Coronary artery disease (CAD) | 37.6% |
| Heart attack | 23.9% |
| Stroke | 16.4% |
| Heart failure | 9.3% |
| Other types of heart disease | 13.1% |

**Genders**

| Gender | Percentage of all heart disease cases |
|---|---|
| Male | 49.3% |
| Female | 50.7% |

**Percentages**

| Age group | Percentage of all heart disease cases |
|---|---|
| 0-19 years | 0.1% |
| 20-39 years | 1.0% |
| 40-59 years | 12.5% |
| 60-79 years | 33.3% |
| 80 years or older | 53.1% |

**Note:** These percentages are based on data from the Centers for Disease Control and Prevention (CDC) and reflect the estimated number of new heart disease cases in 2023.

## How to reduce your risk of heart complications

There are a number of things you can do to reduce your risk of heart disease, including:

* Quitting smoking
* Eating a healthy diet
* Exercising regularly
* Maintaining a healthy weight
* Managing stress
* Controlling blood pressure
* Controlling cholesterol levels
* Controlling blood sugar levels

If you have any concerns about your risk of heart disease, talk to your doctor. They can help you assess your risk and develop a plan to reduce your risk.

## Additional information

It is important to note that heart disease is a complex condition with many risk factors. Some risk factors, such as age and family history, cannot be controlled. However, there are a number of lifestyle choices that can reduce the risk of heart disease.

If you have any questions or concerns about your risk of heart disease, please talk to your doctor. They can help you assess your risk and develop a plan to reduce your risk.

**Comment by shane0** [comment]
https://youtu.be/nQmRw7b3bYI?si=fAja7WHC9dPbXugk

**Comment by shane0** [comment]
Minute timer
https://youtube.com/clip/UgkxPDMd9M_qYSZ6Cjm36BSw39xEoHZF0qpR?si=3fWAwUVhEeHTm-UF

**Comment by shane0** [comment]
https://youtu.be/0X-ldgzfnVI

**Comment by shane0** [comment]
Combine with counting out distraction
Meditation survival game
Concentration vs empty consciousness

**Comment by shane0** [comment]
https://memento-mori-calendar.vercel.app/
